language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/completion/CompletionTest.java

```diff
@@ -116,7 +116,10 @@ public Object[][] testSubset() {
     }
 
     public List<String> skipList() {
-        return new ArrayList<>();
+        List<String> skipList = new ArrayList<>();
+        // Skipping due to https://github.com/ballerina-platform/ballerina-lang/issues/38234
+        skipList.add("completions_inside_generated_idl_client.json");
+        return skipList;
```

Comment:
Shall we do this inside `ClientDeclarationContextTest` WDYT?

Reply:
Addressed in 934532e